src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceV2ApiService.scala

```diff
@@ -0,0 +1,34 @@
+package org.broadinstitute.dsde.firecloud.webservice
+
+import akka.http.scaladsl.server.Route
+
+import org.broadinstitute.dsde.firecloud.FireCloudConfig
+import org.broadinstitute.dsde.firecloud.service.{FireCloudDirectives, FireCloudRequestBuilding}
+import org.broadinstitute.dsde.firecloud.utils.{StandardUserInfoDirectives, StreamingPassthrough}
+
+import scala.concurrent.ExecutionContext
+
+trait WorkspaceV2ApiService
+    extends FireCloudRequestBuilding
+    with FireCloudDirectives
+    with StandardUserInfoDirectives
+    with StreamingPassthrough {
+
+  implicit val executionContext: ExecutionContext
+
+  lazy val rawlsWorkspacesV2Root: String = FireCloudConfig.Rawls.workspacesV2Url
+
+  val workspaceV2Routes: Route =
+    pathPrefix("api" / "workspaces" / "v2") {
+      pathPrefix(Segment / Segment) {
+        (workspaceNamespace, workspaceName) => {
+          val workspaceV2Path = encodeUri(rawlsWorkspacesV2Root + "/%s/%s".format(workspaceNamespace, workspaceName))
+          path("settings") {
+            requireUserInfo() { _ =>
+              streamingPassthrough(workspaceV2Path + "/settings")
+            }
+          }
+        }
+      }
+    }
```

Comment:
I know you're still working on this PR, but what do you think about simplifying this to:
```scala
  val workspaceV2Routes: Route =
    pathPrefix("api" / "workspaces" / "v2") {
      streamingPassthrough(rawlsWorkspacesV2Root)
    }
```

in other words, any request to Orch starting with `/api/workspaces/v2` is proxied to the same request in Rawls.

This would expose a bunch (8?) v2 Rawls APIs which are not currently proxied through Orch - but are publicly accessible anyway. And, it is forward-compatible to support new v2 APIs as they are developed, and it keeps the proxying logic nice and simple.

If you don't want to expose those other APIs, I suggest at least removing the `requireUserInfo()` directive. Since Rawls enforces auth already, no need to enforce it here and then again at the Rawls layer.

Reply:
I committed changes which gut the service and tests to match this setup. Just want to run it by you to make sure I got the idea right.